libraries/AC_Autorotation/AC_Autorotation.h

```diff
@@ -69,17 +89,32 @@ class AC_Autorotation
     float _vel_p;                    // Forward velocity P term.
     float _vel_ff;                   // Forward velocity Feed Forward term.
     float _accel_out;                // Acceleration value used to calculate pitch target.
+	float _entry_sink_rate;
+    float _entry_alt;
+    float _radar_alt;
+    float _rpm_decay;
+    float _flare_entry_speed;
+    float _desired_speed;
+    float _time_to_ground;
+    float _distance_to_ground;
+    float _desired_sink_rate;
+    float _col_mid;
```

Comment:
this is misleading.  we should rename this _coll_zero_thrust.  This should also be changed in the motors class to be consistent.  Autorotation only uses this accessor.

Reply:
Yes, this is a hangover and was not changed when mid col was renamed to zero thrust.

Reply:
Appears that this has been fixed.